### PR TITLE
Fix dataprep

### PR DESCRIPTION
**Pull request by ellieking17**

### CODE REVIEW

File: python/dataprep.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 from keras.utils import to_categorical
 from sklearn.exceptions import DataConversionWarning
 import warnings
+from scipy import sparse
```
Comment (cbaines) [inline python/dataprep.py]: If this is need for the previous commit, I'd squash this commit in to the previous one, or move it before it.

Comment (ellieking17) [reply python/dataprep.py]: I think I just did it in an interactive rebase. Just checking now...

**Comment by ellieking17** [comment]
@cbaines would be good to merge this asap so we can branch off it for our evaluation analysis...

File: python/dataprep.py (hunk 1 above)
Comment (ellieking17) [reply python/dataprep.py]: it's gone. Hurrah. Did fixup instead of squash...

**Pull request merged**